backend/routers/issues.py

```
@@ -247,6 +267,53 @@ async def create_issue(
             linked_issue_id=linked_issue_id
         )
 
+@router.get("/api/issues/{issue_id}", response_model=IssueSummaryResponse)
+def get_issue(issue_id: int, db: Session = Depends(get_db)):
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)